toolkit/tools/scheduler/schedulerutils/buildlist.go

```diff
@@ -101,8 +102,9 @@ func filterLocalPackagesOnly(packageVersionsInConfig []*pkgjson.PackageVer, inpu
 	return
 }
 
-// ReadReservedFilesList updates the list of reserved files from the manifest file passed in.
-func ReadReservedFilesList(path string) (reservedFiles []string, err error) {
+// ReadToolchainPackageManifest updates the list of reserved files (such as toolchain RPMs) from the manifest file passed in.
+// Entries will be returned in the form '<rpm>-<version>-<release>.rpm' with any preceding path removed.
+func ReadToolchainPackageManifest(path string) (reservedFiles []string, err error) {
```

Comment:
Seeing how it's used to read any file with a list of packages, I think the original generic name was more accurate. Especially since the `depsearch.go:isFilteredFile()` function doesn't really care what kind of entries (toolchain packages, a list of groceries, etc.) the list contains.

Reply:
Updated